.github/workflows/ci.yml

```diff
@@ -28,23 +28,31 @@ jobs:
         node-version: '12.x'
     - name: "Install yarn"
       run: npm install -g yarn
-    - name: Setup Lucky
-      run: script/setup
-    - name: Cache Crystal
-      uses: actions/cache@v1
+
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
When setting up the Yarn cache, should it just set the `id` the same as the Crystal cache does?

Reply:
actually id: crystal-cache not need here too, because it not used
it used in config like this https://github.com/skojin/lucky-bug-honeypot/blob/master/.github/workflows/ci.yml#L87
in if command